flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1214,7 +1216,7 @@ public void run() {
 				triggerCheckpoint(System.currentTimeMillis(), true);
 			}
 			catch (Exception e) {
-				LOG.error("Exception while triggering checkpoint.", e);
+				LOG.error("Exception while triggering checkpoint for job " + job + ".", e);
```

Comment:
use `{}` instead? (also applies to other changed lines)

Reply:
I was also thinking about this but decided to keep the style of each individual message (ok, there was no previous style here). I could change all changed lines to adhere the string formats instead of string concatenation if you think, that's better

Reply:
Let's use this opportunity to switch for formats for the changed lines.